data/json/monsters/reptile_amphibian.json

```
@@ -471,5 +471,269 @@
+    "copy-from": "mon_leo_tadpole",
+    "name": { "str": "tadpole" },
+    "upgrades": { "age_grow": 90, "into": "mon_pattern_frog" }
+  },
+  {
+    "id": "mon_bullfrog_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_proxy_tadpole",
+    "name": { "str": "tadpole" },
+    "volume": "240 ml",
+    "weight": "40 g",
+    "hp": 2,
+    "upgrades": { "age_grow": 365, "into": "mon_bullfrog_frog" }
+  },
+  {
+    "id": "mon_strange_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_bullfrog_tadpole",
+    "name": { "str": "tadpole" },
+    "upgrades": { "age_grow": 90, "into": "mon_strange_frog" }
+  },
+  {
+    "id": "mon_giant_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_fish_medium",
+    "description": "A tadpole larger than most bass. It swims a bit like one too.",
+    "name": { "str": "giant tadpole" },
+    "default_faction": "frog",
+    "species": [ "AMPHIBIAN" ],
+    "symbol": "J",
+    "color": "brown",
+    "dissect": "dissect_batrachian_sample_single",
+    "harvest": "mammal_tiny",
+    "vision_night": 30,
+    "upgrades": { "age_grow": 365, "into": "mon_frog_small" },
+    "reproduction": { "baby_egg": "egg_bullfrog_frog", "baby_count": 16, "baby_timer": 60 },
+    "flags": [ "FISHABLE", "SEES", "SMELLS", "HEARS", "SWIMS", "AQUATIC", "WATER_CAMOUFLAGE" ]
+  },
+  {
+    "id": "mon_fowler_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_proxy_tadpole",
+    "name": { "str": "tadpole" },
+    "upgrades": { "age_grow": 30, "into": "mon_fowler_toad" }
+  },
+  {
+    "id": "mon_foul_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_fowler_tadpole",
+    "name": { "str": "tadpole" },
+    "upgrades": { "age_grow": 30, "into": "mon_foul_toad" }
+  },
+  {
+    "id": "mon_freedom_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_proxy_tadpole",
+    "name": { "str": "tadpole" },
+    "upgrades": { "age_grow": 60, "into": "mon_freedom_toad" }
+  },
+  {
+    "id": "mon_odd_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_freedom_tadpole",
+    "name": { "str": "tadpole" },
+    "upgrades": { "age_grow": 60, "into": "mon_odd_toad" }
+  },
+  {
+    "id": "mon_peeper_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_proxy_tadpole",
+    "name": { "str": "tadpole" },
+    "upgrades": { "age_grow": 60, "into": "mon_peeper_frog" }
+  },
+  {
+    "id": "mon_vocal_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_peeper_tadpole",
+    "name": { "str": "tadpole" },
+    "upgrades": { "age_grow": 60, "into": "mon_vocal_frog" }
+  },
+  {
+    "id": "mon_gray_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_proxy_tadpole",
+    "name": { "str": "tadpole" },
+    "upgrades": { "age_grow": 60, "into": "mon_gray_frog" }
+  },
+  {
+    "id": "mon_shift_tadpole",
+    "type": "MONSTER",
+    "copy-from": "mon_gray_tadpole",
+    "name": { "str": "tadpole" },
+    "upgrades": { "age_grow": 60, "into": "mon_shift_frog" }
+  },
+  {
+    "id": "mon_proxy_frog",
```

```suggestion
    "abstract": "mon_proxy_frog",
```
This should be an abstract instead of a full monster

---

Commited, I also changed the tadpole proxy to match.